.github/workflows/certora_recovery.yml

```diff
@@ -0,0 +1,44 @@
+name: certora
+
+on:
+  push:
+    branches:
+      - main
+  pull_request:
+    branches:
+      - main
+
+  workflow_dispatch:
```

Comment:
Is there a reason why we also have "workflow_dispatch" here? I know we also have it in some of our other setups, but unless we want to manually trigger the workflow, I don't think we should.

Reply:
Removed 👍🏾